checkstyle/.scalafmt.conf

```diff
@@ -12,7 +12,10 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-docstrings = JavaDoc
+version = 3.5.9
```

Comment:
So I checked and its actually a requirement of the newer versions of Scalafmt, if you don't have the version there it complains with

```
Caused by: java.util.NoSuchElementException: version [expected 3.5.9 or 3.5.9]: missing
        at metaconfig.Configured.get(Configured.scala:15)
        at com.diffplug.spotless.glue.scalafmt.ScalafmtFormatterFunc.<init>(ScalafmtFormatterFunc.java:46)
        at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
        ... 130 more
```

Not sure what the reasoning behind having to include the include the scalafmt version in the configuration file but aside from some hacky workaround it needs to be done this away.

Reply:
Hmmm... it's a little strange that we now have to specify the version in two places (`gradle/dependencies.gradle` and `checkstyle/.scalafmt.conf`), and it might lead to some headaches when we want to upgrade to a newer version in the future.

If there's no sane way to define the Scalafmt version in a single place (preferably `gradle/dependencies.gradle`), could we leave a comment in the dependencies file noting that the version will also have to be bumped in the Scalafmt configuration file?

Reply:
So I found the release notes where this change was added, i.e. https://github.com/scalameta/scalafmt/releases/tag/v3.1.0 but unfortunately its not very informative as to why the version now has to be explicit.

In any case I have rebased the PR by adding a comment notifying that whenever `scalafmt` is bumped in `dependencies.gradle` that it also needs to be changed in `checkstyle/.scalafmt.conf`

Reply:
Thanks, I think that should be fine.